dispatcher.php

```diff
@@ -39,14 +39,19 @@ public function __construct() {
 		//if requested url starts with api_base_url()
 		if ( false !== strpos( $_SERVER['REQUEST_URI'], get_api_base() ) ) {
 			require_once( 'api/v1/API.php' );
-
-			add_action( 'wp_loaded', array( $this, 'dispatch_api' ) );
+			
+			// Add API dispatch action
```

Comment:
Could you create a new issue here: https://github.com/voceconnect/thermal-api/issues/new and try to explain the problem you are having, I'm having a hard time understanding.